doc/manuals/admin/install.md

```diff
@@ -71,16 +70,6 @@ Upgrade procedure depends on whether the *upgrade path* (i.e. from the installed
 * Upgrading MongoDB version
 * Migrating the data stored in DB (due to a change in the DB data model).
 
-### Upgrading MongoDB version
```

Comment:
This was here from the very beginning but actually is out the scope of Orion documentation... MongoDB official documentation described this information in a much more comprehensive a precise way.

NTC (only informative)